layouts/api/list.html

```diff
@@ -418,14 +418,21 @@ <h3 class="mb-2">Code Example</h3>
 {{range $key, $val := . }}
 -H "Content-Type: {{ $key }}" \{{ end }}
 {{- end -}}
-{{- end -}}
-{{- range $context.action.security -}}
+{{ else }}
+-H "Content-Type: application/json" \{{- end -}}
+{{- with $context.action.security -}}
+{{- range . -}}
 {{- range $key, $val := . -}}
 {{ $authSchema := (index $d.components.securitySchemes $key) }}
 <span class="o">-H</span> <span class="s1">"{{ $authSchema.name }}: <span class="n">${ {{ index $authSchema "x-env-name" }} }</span>"</span>
 {{- end -}}
 {{- end -}}
-{{- if $context.action.requestBody -}} \
+{{- else -}}
+{{ $securitySchemes := $d.components.securitySchemes }}
+<span class="o">-H</span> <span class="s1">"{{ $securitySchemes.apiKeyAuth.name }}: <span class="n">${ {{ index $securitySchemes.apiKeyAuth "x-env-name" }} }</span>"</span> \
+<span class="o">-H</span> <span class="s1">"{{ $securitySchemes.appKeyAuth.name }}: <span class="n">${ {{ index $securitySchemes.appKeyAuth "x-env-name" }} }</span>"</span> 
+{{- end -}}
+{{- if $context.action.requestBody -}}&nbsp;\
```

Comment:
it's inside `<code>`
```suggestion
{{- if $context.action.requestBody -}} \
```

Reply:
yeah i need to use &nbsp; a 'space' isn't rendering. &nsbp; is not rendering in the output though is it? Not even when you copy right?

Reply:
`{{- if $context.action.requestBody }} \` should work